base/prometheus-cloudwatch/cwagent-iam.yaml

```
@@ -0,0 +1,34 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cwagent-prometheus-role
+rules:
+  - apiGroups: [""]
+    resources:
+    - nodes
+    - nodes/proxy
+    - services
+    - endpoints
+    - pods
+    verbs: ["get", "list", "watch"]
```

Minor nit, but could you use the same `verbs` list style that you had for the `ksm-iam.yaml`:
```suggestion
    verbs:
    - get
    - list
    - watch
```

---

I just copy/pasted this part of the config - I will refactor it to match our existing structure - consistency is good!